src/wp-includes/option.php

```diff
@@ -918,20 +927,23 @@ function update_option( $option, $value, $autoload = null ) {
  *
  * @global wpdb $wpdb WordPress database abstraction object.
  *
- * @param string      $option     Name of the option to add. Expected to not be SQL-escaped.
- * @param mixed       $value      Optional. Option value. Must be serializable if non-scalar.
- *                                Expected to not be SQL-escaped.
- * @param string      $deprecated Optional. Description. Not used anymore.
- * @param string|bool $autoload   Optional. Whether to load the option when WordPress starts up.
- *                                Accepts 'yes'|true to enable or 'no'|false to disable.
- *                                Autoloading too many options can lead to performance problems, especially if the
- *                                options are not frequently used. For options which are accessed across several places
- *                                in the frontend, it is recommended to autoload them, by using 'yes'|true.
- *                                For options which are accessed only on few specific URLs, it is recommended
- *                                to not autoload them, by using 'no'|false. Default 'yes'.
- * @return bool True if the option was added, false otherwise.
+ * @param string      $option           Name of the option to add. Expected to not be SQL-escaped.
+ * @param mixed       $value            Optional. Option value. Must be serializable if non-scalar.
+ *                                      Expected to not be SQL-escaped.
+ * @param string      $deprecated       Optional. Description. Not used anymore.
+ * @param bool|null $autoload           Optional. Whether to load the option when WordPress starts up.
+ *                                      Accepts a boolean, or `null` to leave the decision up to default heuristics in WordPress.
+ *                                      For backward compatibility 'yes' and 'no' are also accepted.
+ *                                      Autoloading too many options can lead to performance problems, especially if the
+ *                                      options are not frequently used. For options which are accessed across several places
+ *                                      in the frontend, it is recommended to autoload them, by using 'yes'|true.
+ *                                      For options which are accessed only on few specific URLs, it is recommended
+ *                                      to not autoload them, by using false.
+ *                                      Default is 'null' and this will set to autoload if the value is too large.
+ *
+ * @return bool                         True if the option was added, false otherwise.
```

Comment:
Please fix the indentation:
```suggestion
 * @param string    $option     Name of the option to add. Expected to not be SQL-escaped.
 * @param mixed     $value      Optional. Option value. Must be serializable if non-scalar.
 *                              Expected to not be SQL-escaped.
 * @param string    $deprecated Optional. Description. Not used anymore.
 * @param bool|null $autoload   Optional. Whether to load the option when WordPress starts up.
 *                              Accepts a boolean, or `null` to leave the decision up to default heuristics in WordPress.
 *                              For backward compatibility 'yes' and 'no' are also accepted.
 *                              Autoloading too many options can lead to performance problems, especially if the
 *                              options are not frequently used. For options which are accessed across several places
 *                              in the frontend, it is recommended to autoload them, by using 'yes'|true.
 *                              For options which are accessed only on few specific URLs, it is recommended
 *                              to not autoload them, by using false.
 *                              Default is null, which means WordPress will determine the autoload value.
 *
 * @return bool True if the option was added, false otherwise.
```